docs/source/high_level_overview.rst

```diff
@@ -47,10 +49,25 @@ One important point here is the relation and separation between sample sheets an
 The aim of the sample sheets is to provide a useful computer-readable description of an experiment for processing the data with Bioinformatics workflows or further downstream analyses (e.g. in R or using Jupyter notebooks).
 This document describes a concept for such schemas and an implementation using JSON for the file format.
 
-Of course, sample sheets could also be stored in a RDBMS (e.g., Postgres), with a suitable database schema.
+Of course, sample sheets could also be stored in a RDBMS (Relational database management system, e.g., Postgres), with a suitable database schema.
 Often such databases would be part of a LIMS system.
 A sample sheet could serve as a poor Bioinformatician's LIMS system, if only some meta information for samples and their relation is to be tracked.
 However, the more complex part of such a LIMS system or even a hospital information management system are mostly not of direct relevance to the Bioinformatics analysis.
 
 Rather, the data from a laboratory or hospital IT system can be used to generate biomedical sample sheets.
 BioMed sample sheets further allow linking back from the bio entities, bio samples, test samples, ngs libraries to such systems by storing lists of URLs for each of the previous entities.
+
+-----------------------------------------
```

Comment:
I don't think this section is semantically correct.

Rather, each sheet consists of two levels. The "body" describes the bio entities, bio samples, test samples, and NGS libraries. There are some fixed fields such as pk, secondaryId etc. Then, there is extra information that can be assigned to each object. These fields are not free-form but their structure (data type, e.g., min/max values, regex, choices) is defined in the other part, the "header" or "meta" part.

There is a JSON schema that give some restriction on valid JSON sample sheet files but that's a different animal.

The TSV files are actually "shortcuts" to human-readable and -writeable data formats. Internally, they are converted into the JSON file format.